catroid/src/androidTest/java/org/catrobat/catroid/uitest/content/interaction/UserBrickTest.java

```diff
@@ -545,14 +545,14 @@ public void testBackPackAndUnPackFromDifferentProgrammes() {
 		backPackFirstUserBrickWithContextMenu(DEFAULT_USERBRICK_GROUP_NAME);
 		assertTrue("Userbrick wasn't backpacked!", solo.waitForText(DEFAULT_USERBRICK_GROUP_NAME, 0, TIME_TO_WAIT_BACKPACK));
 
-		UiTestUtils.switchToProgrammBackground(solo, UiTestUtils.PROJECTNAME1, "cat");
+		UiTestUtils.switchToProgrammeBackground(solo, UiTestUtils.PROJECTNAME1, "cat");
```

Comment:
Voting for american english (`program` and `programs`). It's much more consistent with our codebase.
http://dict.leo.org/?lp=ende&search=programm also suggests program for computer programs.